### PR TITLE
Add pucker/bloat modifier

### PR DESCRIPTION
**Pull request by mbasaglia**
Adds a modifier to the specs, it's rather simple to implement and already supported by lottie-web.

### CODE REVIEW

File: schema/shapes/pucker-bloat.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+                    "type": "string",
+                    "const": "pb"
+                },
+                "a": {
+                    "title": "Amount",
+                    "description": "Amount as a percentage",
+                    "$ref": "#/$defs/properties/scalar-property"
+                }
+            },
+            "required": [
```
Comment (fmalita) [inline schema/shapes/pucker-bloat.json:24]: Should `a` be required also?

Comment (mbasaglia) [reply schema/shapes/pucker-bloat.json:24]: it can default to 0

**Review by b-wils — APPROVED** [review]

**Review by fmalita — APPROVED** [review]

**Review by Aidosmf — APPROVED** [review]

**Comment by Aidosmf** [comment]
@mbasaglia should we merge to a develop or version-specific branch?

**Comment by mbasaglia** [comment]
> @mbasaglia should we merge to a develop or version-specific branch?

Not sure, we could also cherry pick on patch branches

**Comment by mbasaglia** [comment]
@cla-bot check

**Pull request merged**